examples/fluid/falling_water_spheres_2d.jl

```diff
@@ -0,0 +1,94 @@
+# In this example two circles of water drop to the floor demonstrating the difference
```

Comment:
1. This example is unstable when I run it on multiple threads. It just continues with a time step < eps.
2. I don't see a difference between surface tension and no surface tension:
![grafik](https://github.com/trixi-framework/TrixiParticles.jl/assets/44124897/616cd786-9755-4b2a-b1ce-ca67f7f079a2)

Reply:
2. There is a difference. For more effect we need to increase the resolution and time the simulation runs.

Reply:
On multiple threads, it freezes, or when I pass `dtmin=eps()`, I get
```
#timesteps:   2100 │ Δt: 5.8212e-05 │ sim. time: 1.6747e-01 (55.823%)  │ run time: 1.1561e+01 s
#timesteps:   2150 │ Δt: 3.6845e-05 │ sim. time: 1.7009e-01 (56.697%)  │ run time: 1.1850e+01 s
#timesteps:   2200 │ Δt: 3.0012e-12 │ sim. time: 1.7114e-01 (57.045%)  │ run time: 1.2212e+01 s
#timesteps:   2250 │ Δt: 3.8658e-16 │ sim. time: 1.7114e-01 (57.045%)  │ run time: 1.2550e+01 s
┌ Warning: dt(2.220446049250313e-16) <= dtmin(2.220446049250313e-16) at t=0.17113580356914368, and step error estimate = 0.9780159000688392. Aborting. There is either an error in your model specification or the true solution is unstable.
└ @ SciMLBase ~/.julia/packages/SciMLBase/NjslX/src/integrator_interface.jl:619
```
Does the same happen for you? Or is it perfectly stable on your machine?

Reply:
I don't run into any issues on my computer. 
Julia 1.10.2
OrdinaryDiffEq v6.76.0